src/validationinterface.cpp

```diff
@@ -44,11 +53,13 @@ void UnregisterAllValidationInterfaces() {
     g_signals.Broadcast.disconnect_all_slots();
     g_signals.Inventory.disconnect_all_slots();
     g_signals.SetBestChain.disconnect_all_slots();
-    g_signals.UpdatedTransaction.disconnect_all_slots();
+    g_signals.BackupWalletAuto.disconnect_all_slots();  //MVHF-BU
+	g_signals.UpdatedTransaction.disconnect_all_slots();
```

Comment:
not sure if github is formatting this weird, or if indentation is different than rest